src/proto/Harmony.proto

```diff
@@ -45,3 +45,92 @@ message SigningOutput {
     bytes r = 3;
     bytes s = 4;
 }
+
+message StakingTransactionInput {
```

Comment:
can we reuse SigningInput and SigningOutput? take a look at `Waves.proto`

Reply:
SigningOutput could be reused as is. However, SigningInput and StakingSiningInput have some common fields, do you recommend merging the fields?

Reply:
Fixed. Followed Waves.proto style.